crates/rpc/src/handlers/state.rs

```diff
@@ -1,10 +1,35 @@
+use alloy_primitives::FixedBytes;
 use ream_consensus::deneb::beacon_state::BeaconState;
-use ream_storage::{db::ReamDB, tables::Table};
+use ream_storage::{db::ReamDB, tables::{ Field, Table}};
+use warp::{
+    http::status::StatusCode, 
+    reject::Rejection, 
+    reply::{with_status, Reply}
+};
 
 use crate::types::{errors::ApiError, id::ID};
 
+use super::BeaconResponse;
+
 pub async fn get_state_from_id(state_id: ID, db: &ReamDB) -> Result<BeaconState, ApiError> {
     let block_root = match state_id {
+        ID::Named(ref name) => match name.as_str() {
+            "head" => {Ok(Some(FixedBytes::new([0; 32])))},
```

Comment:
Hey @KolbyML could you give some input here.
For `state_id` I am adding the variants `head`, `justified`, `finalized`, `genesis` to have this endpoint return the corresponding BeaconState.

I've just added some placeholders for the ones I am not sure how to get yet.
Here how should I be getting the `head` block_root?

Reply:
> Here how should I be getting the head block_root?

I think it is fine if we return an error for now